src/oc_erchef/apps/chef_index/test/chef_index_batch_tests.erl

```diff
@@ -119,3 +126,14 @@ wait_for_started() ->
         started ->
             ok
     end.
+
+%% Note that this cannot work if the item added is bigger than max_size -- these
+%% will be flushed immediately, and this _could_ have happened before we started
+%% waiting/looping.
+wait_for_added() ->
+    case chef_index_batch:status() of
+        #{ item_queue := [] } ->
+            timer:sleep(10),
```

Comment:
This was the intent.  We believe the race is literally between two *very fast most of the time* calls, so we didn't want to slow tests down too much.  I cold see going to 100ms though.